### ISSUE TITLE
How to arrange items into 3+ cols? default 2 is few for labels in large numbers

### ISSUE THREAD

**Issue by YustinLau** [gurusabarish/hugo-profile#248]
Hello! Thx for the marvelous theme! A little question for customizing css:

<img width="1745" height="604" alt="Image" src="https://github.com/user-attachments/assets/279e2b44-8777-4782-91c3-a8d2953f45ae" />

as shown above, I need to list many items(12+) so more columns needed instead of 2. already tried to modify list.css and other css files but not help. Is it possible to achieve arrangement in 3+ cols?

i.e. in the screenshot example:

|C/C++ |Python |SQL|
|:--:|:--:|:--:|
|Linux Kernel|Flink|Kafka|
|TensorFlow|Pytorch|Machine Learning|
|...|...|...|
